### DISCUSSION TITLE
Cycling routes is not disappear after disabling

### DISCUSSION THREAD

**Discussion by anastasiia936** [osmandapp/OsmAnd#12454] [General]
### How to reproduce?
Enable and disable the Cycling route.

![image](https://user-images.githubusercontent.com/80623554/129032547-169bdf97-6804-40de-a14c-bd43885a84a4.png)
![image](https://user-images.githubusercontent.com/80623554/129032827-c13e8999-7fa0-4b60-9fef-de4d49cbcf4b.png)
![image](https://user-images.githubusercontent.com/80623554/129032848-495d1fcf-0878-4e77-a4f5-8247ba9a2775.png)

**Comment by scaidermern** [comment]
Can confirm. The bicycle routes stay visible after disabling them (the color changes a little after moving/zooming the map). However it looks like this only happens when the bicycle route type is set to node network.

**Comment by vshcherb** [comment]
Well yes, it is done on purpose for bicycle mode. It was introduced much earlier than all options appeared. And it is a good question now should we disable it or not. Will transform it into discussion cause it's not clear now.

**Reply by davidq1973** [reply]
It does not only occur in bicyclemode.
It would be nice if you could toggle between enable and disable as the option suggests it does.

**Reply by Nes51** [reply]
I agree with David. It would be a lot easier and user friendlier if it is possible to disable Bicycle Routes by just using the toggle in the Map Settings menu.
However there is a (more complicated way to disable:
- tap the menu dots of Bicycle Routes
- switch from Nodes to Routes
- turn off with the toggle
- after about 10 to 12 seconds the bicycle routes are disabled
![Screenshot_20210812-195046_OsmAnd~](https://user-images.githubusercontent.com/88850898/129246076-f211dc99-07bf-45a8-ae53-1a39678f56c4.jpg)

**Reply by vshcherb** [reply]
The issue with UX / Discoverability, so user won't see cycle routes once he turns on the bicycle mode.

**Reply by Nes51** [reply]
?????
Please explain what you mean. This reply is complete "secret language" to me.
The issue, and subsequently request, is: Why can I not turn off the Nodes Network Routes by simply using the switch in "Show Routes"?
